### PR TITLE
clustering, aj villegas

### PR DESCRIPTION
**Pull request by theAjforce**

### CODE REVIEW

File: clustering 2/clustering.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  s_score.append(silhouette_score(x_scaled,km.labels_))
+  c_score.append(calinski_harabasz_score(x_scaled,km.labels_))
+
+plt.plot(range(2,11),inertia)
+plt.plot(range(2,11),s_score)
+plt.plot(range(2,11),c_score)
+km = KMeans(n_clusters=3,n_init=20)
+km.fit(x_scaled)
+plt.scatter(x_scaled['13495'],x_scaled['111'],c=km.labels_)
+plt.scatter(km.cluster_centers_[:,0],km.cluster_centers_[:,1],marker='x',c='r')
```
Comment (Svangorden13) [inline clustering 2/clustering.py:31]: Need to include boundaries in plot (hint: google voronoi diagram)

File: clustering 2/clustering.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+c_score = []
+
+for k in range(2,11):
+  km = KMeans(n_clusters=k,n_init=20)
+  km.fit(x_scaled)
+  inertia.append(km.inertia_)
+  s_score.append(silhouette_score(x_scaled,km.labels_))
+  c_score.append(calinski_harabasz_score(x_scaled,km.labels_))
+
+plt.plot(range(2,11),inertia)
```
Comment (Svangorden13) [inline clustering 2/clustering.py:25]: X- and y-axis labels and legends would be helpful in plots

File: clustering 2/clustering.py [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+dbs = DBSCAN(eps=0.125,min_samples=5)
+dbs.fit(x_scaled)
+plt.scatter(x_scaled['13495'],x_scaled['111'],c=dbs.labels_)
+#3.
+mlb_df = pd.read_csv('/content/mlb_batting_cleaned.csv')
+
+def nearest_players(player):
+  df = mlb_df[mlb_df['Name'] == player]
+  df = df.drop(['Name','Tm','Lg'],axis=1)
+  x = mlb_df.drop(['Name','Tm','Lg'],axis=1)
```
Comment (Svangorden13) [inline clustering 2/clustering.py:42]: Isn't this the same as df?

File: clustering 2/clustering.py [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+  scaler = MinMaxScaler()
+  x_scaled = pd.DataFrame(scaler.fit_transform(x),columns=x.columns)
+  nbrs = NearestNeighbors(n_neighbors=3)
+  nbrs.fit(x_scaled)
```
Comment (Svangorden13) [inline clustering 2/clustering.py:46]: This should be fit on the full dataframe

File: clustering 2/clustering.py [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+  _, indices = nbrs.kneighbors(df)
```
Comment (Svangorden13) [inline clustering 2/clustering.py:47]: This should look at the single player

**Review by Svangorden13 — CHANGES_REQUESTED** [review]
